.eslintrc.js

```diff
@@ -1,9 +1,9 @@
-// eslint-disable-next-line no-undef
 module.exports = {
-  "ignorePatterns": ["node_modules", "dist"],
+  "ignorePatterns": ["node_modules", "dist", ".eslintrc.js", "spicetify.d.ts"],
   "env": {
     "browser": true,
-    "es2021": true,
+    "commonjs": true,
```

Comment:
Do we need commonjs? We don't use `require()`

Reply:
It is not only for "require"

Reply:
But I can remove it if you want